sematic/ui/src/components/GitInfo.tsx

```diff
@@ -0,0 +1,128 @@
+import { useCallback, useState } from "react";
+import { Box, ButtonBase, Link, Tooltip, Typography, useTheme } from "@mui/material";
+import { ContentCopy, OpenInNew } from "@mui/icons-material";
+import PostAddIcon from '@mui/icons-material/PostAdd';
+import { RiGitBranchLine, RiGitCommitLine, RiGitRepositoryCommitsLine } from "react-icons/ri";
+import { FaGitSquare } from "react-icons/fa";
+import { Resolution } from "../Models";
+
+/**
+ * Turns the following remote formats:
+ * - git@gihub.com:sematic-ai/sematic.git
+ * - https://github.com/sematic-ai/sematic.git
+ * into clickable links:
+ * - https://github.com/sematic-ai/sematic/tree/<branch>
+ * - https://github.com/sematic-ai/sematic/commit/<SHA>
+ */
+function makeGithubLink(remote: string, path: string) {
+  let domain = remote
+    .replace(/^(git@)|(https:\/\/)/, "")
+    .replace(/(\.git)$/, "")
+    .replace(/:/, "/")
+  return "https://" + domain + "/" + path
+}
+
+function GitInfo(props: {
+    text: string, tooltip: string, remote: string, path: string, children?: any
+}) {
+
+  const [content, setContent] = useState(props.text);
+
+  const copy = useCallback(() => {
+    navigator.clipboard.writeText(props.text);
+    // avoid temporary resizing by preserving the initial text length
+    // by using non-breaking spaces
+    setContent("Copied".padStart(props.text.length, " "));
+    setTimeout(() => setContent(props.text), 1000);
+  }, [props.text]);
+
+  return (
+
+    <Typography color="GrayText" component="span">
+      <Tooltip title={props.tooltip}>
+        <ButtonBase onClick={copy}>
+          {props.children}: <code>{content}</code>
```

Comment:
Can you remove the colon `:`?
Also, how about this: We set the link around `content`, and get rid of the OpenInNew icon. Removes a bit of clutter.

Reply:
Done. Done.
<img width="240" alt="image" src="https://user-images.githubusercontent.com/1894533/194944497-14a35c5b-ef6e-45ba-8d11-5793496a2da9.png">